packages/web-pkg/src/composables/actions/files/useFileActionsCopyPermanentLink.ts

```diff
@@ -15,23 +18,29 @@ export const useFileActionsCopyPermanentLink = () => {
       await copyToClipboard(url)
       showMessage({ title: $gettext('The link has been copied to your clipboard.') })
     } catch (e) {
-      console.error(e)
       showErrorMessage({
         title: $gettext('Copy link failed'),
         errors: [e]
       })
     }
   }
-
+  interface FileActionOptionsWithEvent extends FileActionOptions<Resource> {
+    event?: MouseEvent
+  }
```

Comment:
I would move this to the file where the `FileActionOptions` interface is located.